src/groq/index.ts

```diff
@@ -94,6 +95,13 @@ export default class GroqChat {
         temperature,
       );
 
+      if (tokenUsageInformation) {
+        if (!chatCompletion.usage) {
+          throw new Error('Token usage information is not available.');
+        }
+        this.saveTokenUsageInfo(chatCompletion?.usage);
```

Comment:
suggestion: At the moment, we display the `Completion, Usage, and Total` tokens for each individual file processed. This can be cumbersome to read when `optimizeit` is processing several files at once. I'll leave it up to you, but, do you think a better approach here is to `accumulate` them and display the `completion, usage, and total` for all of the files processed instead of 1 by 1?